### ISSUE TITLE
Always create new calendar note in edit mode

### ISSUE THREAD

**Issue by kwiberg** [liamcain/obsidian-calendar-plugin#196]
**Is your feature request related to a problem? Please describe.**

When I click on a day in the calendar widget to create a new daily note, it’s not immediately obvious whether it’s in edit or view mode, since there’s no text in it yet. So I have to start typing, see if it works, and press Ctrl-E if not. (It seems like the new note inherits the edit/view state of whatever note was in that frame previously.)

**Describe the solution you'd like**

Always create new daily notes in edit mode.

**Describe alternatives you've considered**

A config setting? I can't see why anyone would want new notes in view mode, though.

**Comment by Hamezii** [comment]
In the Editor tab in the Obsidian options menu, there's a "Default new pane mode" property, which you can set to "Editor" if you want every new note/pane you open to start in edit mode instead of view mode. 

If you want a "default view mode" specifically for when you create a new note, as opposed to opening a new note pane, that feels more like it should be a base Obsidian option request, or for a separate QOL plugin

**Comment by kwiberg** [comment]
Yes, Obsidian has a toggle for whether new panes should start in edit or view mode, but that one doesn't care whether a note is empty or not, so it doesn't help in this case.

As it turns out, Obsidian already ensures that new and empty notes are always opened in edit mode, regardless of what that toggle button would otherwise dictate: https://forum.obsidian.md/t/always-create-new-note-in-edit-mode/22296

So what I'm asking for is just that the calendar plugin behave like Obsidian proper, and always open new and empty notes in edit mode.

**Comment by Mansarde** [comment]
It's been two years, but I've only recently discovered Obsidian and had the same feature-wish regarding the Calendar plugin.
I played around a bit and ultimately ended up implementing it myself. 😁

Since the maintainer hasn't merged any PR since last year, I've also built a release on my fork that you can use until the PR is merged: https://github.com/Mansarde/obsidian-calendar-plugin/releases/tag/1.5.11

<sub>(Alternatively you could use [Plugins Galore](https://github.com/plugins-galore/obsidian-plugins-galore) to install it directly via the repo URL.)</sub>
